src/services/iotHubManagerService.js

```diff
@@ -46,4 +48,14 @@ export class IoTHubManagerService {
     return HttpClient.delete(`${ENDPOINT}devices/${id}`)
       .map(() => ({ deletedDeviceId: id }));
   }
+
+  /** Returns the account's device group filters */
+  static getDeviceProperties() {
+    return Observable
+      .forkJoin(
+        HttpClient.get(`${ENDPOINT}deviceproperties`),
+        HttpClient.get(`${Config.serviceUrls.deviceSimulation}devicemodelproperties`)
```

Comment:
the correct URL is camelCase: "deviceModelProperties"

Reply:
I've refereed few other URLs in our services, we don't use camel case. Besides it shouldn't matter.

Reply:
@dluc camelcase doesen't work with Java Play  framework